pkgs/applications/networking/cluster/rke2/README.md

```diff
@@ -1,15 +1,19 @@
 # RKE2 Version
 
-RKE2, Kubernetes, and other clustered software has the property of not being able to update atomically. Most software in nixpkgs, like for example bash, can be updated as part of a `nixos-rebuild switch` without having to worry about the old and the new bash interacting in some way.
+RKE2, Kubernetes, and other clustered software has the property of not being able to update atomically.
+Most software in nixpkgs, like for example bash, can be updated as part of a `nixos-rebuild switch`
+without having to worry about the old and the new bash interacting in some way.
 
 > [!NOTE]
 > Upgrade the server nodes first, one at a time. Once all servers have been upgraded, you may then upgrade agent nodes.
 
 ## Release Channels
 
-RKE2 has there own release channels, which are: `stable`, `latest` and `testing`.
+RKE2 has three main release channels, which are: `stable`, `latest` and `testing`.
 
-The `stable` channel is the default channel and is recommended for production use. The `latest` channel is the latest stable release. The `testing` channel is the latest release, including pre-releases.
+The `stable` channel is the default channel and is recommended for production use.
+The `latest` channel is the latest stable release.
+The `testing` channel is the latest release, including pre-releases.
```

Comment:
```suggestion
The `stable` channel is the default channel and is recommended for production use.
The `latest` channel is the latest release.
The `testing` channel is the latest release, including pre-releases.
```
It might be confusing if we call the `latest` channel 'stable' as we also have a channel called `stable`.

Reply:
Agree!